### PR TITLE
104 export hs building as seed

### PR DESCRIPTION
**Pull request by elenagensch**
- [x] Adds rake task  export:export_to_seeds for
-  - [x]  retrieving Point, PointOfInterest, Polyline, Wall, Room and Building from db
-  - [x]  logging corresponding create statements

 - [x] seed building data (add logged statements to seeds.rb)
 
image

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -48,12 +36,87 @@
   [ "H-2.3", floor, "Bachelorprojekt Baudisch"],
```
Comment (SebastianWagner2) [inline db/seeds.rb:36]: Please remove the room list, the rooms in it end up being nonsensical (no outer line)

File: db/seeds.rb [hunk 2]
```diff
@@ -48,12 +36,87 @@
...
   [ "A-1.15", floor2, "A-1.15"]
 ]
+hs_rooms = [
+  [floor, "HS 3", 1],
+  [floor, "Passage", 2],
+  [floor, "HS 2", 3],
+  [floor, "Foyer", 4],
+  [floor, "R1", 5],
+  [floor, "R2", 6],
+  [floor, "R3", 7],
```
Comment (SebastianWagner2) [inline db/seeds.rb:46]:
```suggestion
  [floor, "Passage", 2],
  [floor, "HS 2", 3],
  [floor2, "Foyer", 4],
  [floor2, "R1", 5],
  [floor, "R2", 6],
  [floor, "R3", 7],
```
Please use both first and second floor (we need to test smth)

File: db/seeds.rb [hunk 3]
```diff
@@ -88,18 +151,35 @@
...
 end
 
+hs_rooms.each do |room|
+  outer_shape = Polyline.find(room[2]) if room[2].present?
+  Room.create(floor: room[0], full_name: room[1], outer_shape: outer_shape)
+end
+
 room_type_list.each do |room_type|
   RoomType.create(name: room_type[0])
 end
```
Comment (SebastianWagner2) [inline db/seeds.rb:177]:
```suggestion
point_list.each do |point|
  Point.create(point)
end

point_of_interest_list.each do |point_of_interest|
  PointOfInterest.create(point_of_interest)
end

polyline_list.each do |polyline_points|
  Polyline.create(points: polyline_points.map { |point_id| Point.find(point_id) })
end

bundle = person_collection.zip hs_rooms

bundle.each do |person, room|
  outer_shape = Polyline.find(room[2]) if room[2].present?
  Room.create(number: room[0], floor: room[1], full_name: room[2], people: [person])
end

room_type_list.each do |room_type|
  RoomType.create(name: room_type[0])
end
```

Comment (SebastianWagner2) [reply db/seeds.rb:177]: Fixed this not to use rooms_list anymore (use hs_rooms instead)

Am not sure about lines 166 through 169, looks to me like the same room may be created several times (not sure though)

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]
Thanks for updating the seeding to our changes :)

Co-authored-by: benjamin-frost <bfrost.schule@gmail.com>
Co-authored-by: LeonHermann322 <leon.hermann@student.hpi.uni-potsdam.de>
Co-authored-by: Toalaah <samuel.kunst@gmail.com>
Co-authored-by: kolioOtSofia <n.genchev@hotmail.com>
Co-authored-by: Tinus Lorvalds <torvalds@linux-foundation.org>

File: db/seeds.rb [hunk 4]
```diff
@@ -35,25 +25,96 @@
...
-  [ "A-1.15", floor2, "A-1.15"]
+hs_rooms = [
+  ['D-E.3', floor, "HS 3", 1],
+  ['D-E.5', floor, "Passage", 2],
+  ['D-E.2', floor, "HS 2", 3],
+  ['D-E.6', floor, "HS Foyer", 4],
+  ['D-E.4', floor, "HS Anrichte", 5],
+  ['D-E.7', floor, "HS Lager", 6],
+  ['D-E.8', floor, "HS Elektro", 7],
+  ['D-K.1', floor2, "ping pong", 4],
```
Comment (cedric-lorenz) [inline db/seeds.rb:44]:
```suggestion
  ['D-K.1', floor2, "Ping Pong", 4],
```

**Review by cedric-lorenz — APPROVED** [review]

**Review by cedric-lorenz — APPROVED** [review]

File: db/seeds.rb (hunk 4 above)
Comment (elenagensch) [reply db/seeds.rb]: 👍

**Pull request merged**